Resources/config/services.xml

```diff
@@ -5,7 +5,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <parameters>
-        <parameter key="admingenerator.form.upload_collection_listener.class">Admingenerator\FormExtensionsBundle\EventListener\UploadCollectionListener</parameter>
+        <parameter key="admingenerator.form.upload_collection_listener.class">Admingenerator\FormExtensionsBundle\EventListener\CollectionUploadListener</parameter>
```

Comment:
Hi @pjam 

Thanks for your PR.

To be consistent with other renaming, could you please rename the listener to `UploadCollectionListener` rather than changing the service here?

Thanks!